### DISCUSSION TITLE
Increase hardware resources to reduce search time?

### DISCUSSION THREAD

**Discussion by vuongdanghuy** [google/or-tools#3580] [Routing (and legacy CP) questions]
Hi everyone.

I have one question about ortools. Currently i'm using ortools to solve VRPTW problem with GUIDED_LOCAL_SEARCH as local search metaheuristic and 2s timeout. 

The question is, if i want to reduce timeout of local search to 1s, can i increase hardware resources like double the number of CPUs to keep the solution as good as possible like the one i got with 2s timeout?

Thank u very much.

**Comment by Mizux** [comment]
not sure routing is multi-thread so you won't gain. Usualy we run a model with various initial solution in parallel then keep the best solution among the processes.

**Reply by vuongdanghuy** [reply]
Thanks for your response. 

In my case, currently i have only 1 initial solution that is a list of node already assigned for that vehicle so i think it's not possible.

**Comment by gregy4** [comment]
You didn’t say what hardware you are using today. I was surprised by performance of new zen4 in both single and multi-thread tasks, in my case speedup was more than I expected based on general performance tests. I didn’t tested vrp but another constraint solver so results can be similar.
